### DISCUSSION TITLE
Shared modals - How to ?

### DISCUSSION THREAD

**Discussion by joffreypersia** [livewire/livewire#8284] [1. Help]
Hi everyone,

I’m working on a large-scale application with a back-end developer friend of mine. We’re facing a challenge with implementing a shared modal that can be triggered by different parts of our bulk action table and from the table row.

We need this modal for various purposes, such as confirming item deletions or displaying additional data to the user, like selecting a tag to apply to the items.

We're considering whether it should be a Livewire (LW) modal that loads a specific Blade view. However, we’re unsure about the data transmission process since a Livewire modal would operate as an island, completely isolated from other variables on the page. This approach could also help us reduce the number of Blade components processed in the table rows.

I've gone through Laracasts and Caleb Porzio's tutorials, but haven’t found anything related to this issue.

Do you have any suggestions on how to approach this? Any advice would be greatly appreciated.

Thank you!

**Comment by joshhanley** [comment]
@joffreypersia Have a look at https://github.com/wire-elements/modal as it should do what you're after.
